docs/src/model_creation/examples/basic_CRN_library.md

```diff
@@ -1,4 +1,22 @@
 # [Library of Basic Chemical Reaction Network Models](@id basic_CRN_library)
+```@raw html
+<details><summary><strong>Tutorial environment set-up</strong></summary>
+```
+The following code sets up an environment for running this tutorial.
```

Comment:
Thinking of making part of the test a link to https://docs.sciml.ai/Catalyst/stable/introduction_to_catalyst/catalyst_for_new_julia_users/#catalyst_for_new_julia_users_packages, for anyone who wants clarifications.